### PR TITLE
Fix: MFA Device recognition regression

### PR DESCRIPTION
**Pull request by pcmxgti**

### CODE REVIEW

**Comment by pcmxgti** [comment]
@opis-mark This should fix #144 if you run Tokendito with `--use-device-token` twice in a row. Alternatively, you can pre-save your device token under `okta_device_token` in the config file. I'll do a bit more testing over the next few days before making this an official pull request.

**Comment by ruhulio** [comment]
@pcmxgti Thanks for putting up this PR. After updating to `2.3.0` earlier today, I had noticed that the `use_device_token` read/write changes had been lost when merging with the `OIE` and `cmd_interface` changes. Let me know if there is anything I can help with around this.

**Comment by pcmxgti** [comment]
> @pcmxgti Thanks for putting up this PR. After updating to `2.3.0` earlier today, I had noticed that the `use_device_token` read/write changes had been lost when merging with the `OIE` and `cmd_interface` changes. Let me know if there is anything I can help with around this.

No worries! It turns out we also have to have a user agent that Okta likes for this to work correctly. We should be done with our testing shortly.

File: tests/unit/test_okta.py [hunk 1]
```diff
@@ -585,7 +590,7 @@ def test_step_up_authenticate(mocker):
     assert okta.step_up_authenticate(pytest_config, state_token) is False
 
 
-def test_local_auth(mocker):
+def test_local_authicate(mocker):
```
Comment (ruhulio) [inline tests/unit/test_okta.py:593]: 🐝 Spelling…
```suggestion
def test_local_authenticate(mocker):
```

Comment (pcmxgti) [reply tests/unit/test_okta.py]: Nice catch! fixed.

**Review by sevignyj — APPROVED** [review]

**Pull request merged**